include/mxnet/lib_api.h

```diff
@@ -35,6 +35,7 @@
 #include <string>
 #include <iostream>
 #include <utility>
+#include <exception>
```

Comment:
`#include <stdexcept>`

`std::runtime_error` is in `stdexcept`
https://zh.cppreference.com/w/cpp/error/runtime_error

Reply:
Done